commercetools.Sdk/IntegrationTests/commercetools.Sdk.IntegrationTests/Carts/CartsFixture.cs

```diff
@@ -38,6 +40,13 @@ public static CartDraft DefaultCartDraft(CartDraft cartDraft)
             return cartDraft;
         }
 
+        public static CartDraft DefaultCartDraftInStore(CartDraft cartDraft, ResourceIdentifier<Store> store)
```

Comment:
Could this be changed to a IReferenceable?

Reply:
yes, it's solved now :thumbsup: